airbyte-webapp/src/packages/cloud/components/experiments/SpeedyConnection/SpeedyConnectionBanner/SpeedyConnectionBanner.tsx

```diff
@@ -44,7 +35,6 @@ export const SpeedyConnectionBanner = () => {
             timer: () => <CountDownTimer expiredOfferDate={expiredOfferDate} />,
```

Comment:
Just a note that I noticed while testing: I'd suggest to remove the seconds from this counter. Having something ticking down every second of the top of the page feels really destracting from what you want to do on the page.